lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -256,7 +262,7 @@ <h3 class="error-msgs" id="error-msg"></h3>
          }
          if($('#invoice_number').val() == "") {
                      $('#error-msg').attr('class','error-msgs')
-                     $('#error-msg').html("${_('The Invoice Number field cannot be empty.') | n, h, js_escaped_string}").show();
+                    $('#error-msg').text("${_('The Invoice Number field cannot be empty.') | n, js_escaped_string}").show();
```

Comment:
It is needed.  The price we pay for embedding JavaScript in Mako that is more complex than just passing data through to a JavaScript file.